Target a single file for `--in` and `--out` (#322)
scalameta/mdoc: docs/installation.md

+  --in mydocs
```

> You can also target an individual file by passing it to `--in`.

Reviewer: ```suggestion
The `--in` flag doesn't have to be a directory, it also supports individual files.
```

+  --site.SCALA_VERSION @SCALA_VERSION@
```

### Customize output directory

Use `--out` to customize where your markdown sources are generated, by default
the `out/` directory is used.

```diff
 coursier launch org.scalameta:mdoc_@SCALA_BINARY_VERSION@:@VERSION@ -- \
+  --out target/docs
```

### Generate single output file instead of directory

The `--out` flag doesn't have to be a directory when the `--in` argument is a
regular file, it can also be an individual file.

```diff
 coursier launch org.scalameta:mdoc_@SCALA_BINARY_VERSION@:@VERSION@ -- \
+  --in readme.template.md \
+  --out readme.md
```

### Process multiple input directories and files

> Repeat the `--in` and `--out` arguments to process multiple directories and

Author: 👍
